docs/model_developers/modules.rst

```diff
@@ -357,19 +357,19 @@ By following the previous steps, the Inmanta server will be able to install modu
 Inter-module dependencies
 #########################
 
-The plugins code of a module mod-a can have a dependency on the plugins code of another V2 module mod-b. When doing this,
+The plugins code of a module ``mod-a`` can have a dependency on the plugins code of another V2 module ``mod-b``. When doing this,
```

Comment:
I think we can remove this entire section. We can no longer run into this issue because we create a new executor process when the code for that executor changes.